README.md

```diff
@@ -48,18 +48,78 @@ built-in help system:
 travis-worker --help
 ```
 
-### Configuring the requested provider
+
+## Development: Running Travis Worker locally
+
+This section is for anyone wishing to contribute code to Worker. The code
+itself _should_ have godoc-compatible docs (which can be viewed on godoc.org:
+<https://godoc.org/github.com/travis-ci/worker>), this is mainly a higher-level
+overview of the code.
+
+### Environment
+
+Ensure you've defined the necessary environment variables (see `.example.env`).
+
+### Pull Docker images
+
+    $ docker pull travisci/ci-amethyst:packer-1504724461
+    $ docker tag travisci/ci-amethyst:packer-1504724461 travis:default
```

Comment:
Please use triple backtick gates instead of legacy indented code example style 😁